sources/lib/api/gcp/bigquery/_table.py

```diff
@@ -627,7 +627,7 @@ def to_dataframe(self, start_row=0, max_rows=None):
 
     # Need to reorder the dataframe to preserve column ordering
     ordered_fields = [field.name for field in self.schema]
-    return df[ordered_fields]
+    return df[ordered_fields] if df else None
```

Comment:
Wouldn't an empty DataFrame be a better choice... so the user doesn't have to check for None in notebook code?